samples/bluetooth/mesh/chat/prj.conf

```diff
@@ -10,7 +10,7 @@ CONFIG_NCS_SAMPLES_DEFAULTS=y
 CONFIG_LOG_MODE_DEFERRED=y
 
 # General configuration
-CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1800
```

Comment:
We got an agreement to add a reason as a comment if the default value is changed to something custom.
Does it have the sense to add it here?

Reply:
here and below (to not generate trash from the comments)

Reply:
This should not be needed at all, default is 2048 (nrf/subsys/bluetooth/controller/Kconfig:
```
# SYSTEM_WORKQUEUE_STACK_SIZE is declared in Zephyr and also here for a second
# time, because Central side Bluetooth sample needs a different workqueue size
# when the SoftDevice Controller is used.
config SYSTEM_WORKQUEUE_STACK_SIZE
	int
	default 2048 if BT_LL_SOFTDEVICE
```
)

Reply:
I reduced it to make chat sample fit on 52832. After adding the mesh settings thread it overflowed the RAM. I can add a comment decribing this.

Reply:
Hm... did you try to configure publication over the gatt with this amount of memory?

Reply:
Just disable FLASH_SHELL instead. It will be enabled here because FLASH && SHELL and is "more obviously not needed" in my opinion at least. And saves you 8k instead of 200 bytes.